docs/kratos/manage-identities/25_import-user-accounts-identities.mdx

```diff
@@ -281,6 +281,61 @@ In that case result of the `salting` process would be:
 salt123--ory123
 ```
 
+</TabItem>
+
+<TabItem value="md5" label="MD5">
```

Comment:
```suggestion
<TabItem value="md5" label="MD5">
```

this needs to be unique, maybe salted-sha ?

Reply:
Ah yeah, copy paste error